addons/io_hubs_addon/components/ui.py

```diff
@@ -142,6 +145,271 @@ def draw(self, context):
         draw_components_list(self, context)
 
 
+def export_scene():
+    try:
+        import os
+        import sys
+        extension = '.glb'
+        output_dir = bpy.app.tempdir
+        if not os.path.exists(output_dir):
+            os.makedirs(output_dir)
+        args = {
+            # Settings from "Remember Export Settings"
+            **dict(bpy.context.scene.get('glTF2ExportSettings', {})),
+
+            'export_format': ('GLB' if extension == '.glb' else 'GLTF_SEPARATE'),
+            'filepath': os.path.join(bpy.app.tempdir, EXPORT_TMP_FILE_NAME),
+            'export_cameras': True,
+            'export_lights': True,
+            'export_extras': True,
+            'use_visible': True,
+            'export_apply': True
```

Comment:
Actually this is also left over from a previous version. My idea was to totally rely on the exporter settings. At least for the initial version. After that we can look for a way to maybe expose the most common export settings in the Hubs panel to override the glTF exporter settings if explicitly requested by the user. For now I'll remove this overrides.

Reply:
I did some checking into embedding the glTF export panel and sadly I don't think it's an option.  I don't know if maybe we want to have a toggle between overriding settings and just using the saved glTF properties, but I think if we're going to have overrides then it would be nice to have all the `Limit to` options (although that depends on Blender version).  Aside from those, the `Rest & Ranges` options could affect scene preview, and I think `Sampling Animations` should be forced off (have it there, but greyed out and disabled, or maybe just a label that says sample animations disabled?) because it can cause problems with animations and increases the file size.

I'm not really sure what to say about this to be honest.  I don't think anything will fit perfectly, and what you have currently, technically, allows advanced users to squeak by and accomplish all possible exports, but it's not super intuitive.

Reply:
This is generally good, thanks, but I think the fact we are forcing no sampling of animations should be conveyed to the user, along with our reasoning (in a tooltip I guess), especially since it defaults to on in the regular glTF export.  Also, the use_active_scene property was only introduced in Blender 3.2, but yes,having use_active_scene on is useful, but again I think we want to let the user know that because if they do have multiple scenes, but don't know to check that button in the exporter, then I believe it will work for the debugger, but not when they export to upload to Spoke (because they didn't know to check that export option).